common/templates/course_modes/choose.html

```diff
@@ -82,9 +82,9 @@ <h4 class="title">${_("Pursue a Verified Certificate")}</h4>
                                                         ${_("Benefits of a verified Certificate")}
                                                     </h4>
                                                     <ul>
-                                                        <li><b>${_("Official")}: </b>${_("Receive an instructor-signed certificate with the institution's logo")}</li>
-                                                        <li><b>${_("Easily sharable")}: </b>${_("Add the certificate to your CV or resume, or post it directly on LinkedIn")}</li>
-                                                        <li><b>${_("Motivating")}: </b>${_("Give yourself an additional incentive to complete the course")}</li>
+                                                        <li><b>${_("Official: {b_tag_end}Receive an instructor-signed certificate with the institution's logo").format(b_tag_end='</b>')}</li>
```

Comment:
Let's pull the `<b>` into this string as well. It seems odd to just have the end tag but not the start. (for all 3)